app/src/main/res/values/styles_aptoide_text_view.xml

```diff
@@ -224,4 +223,72 @@
     <item name="android:textColor">@color/youtube</item>
   </style>
 
+  <!-- Theme Color -->
+
+  <style name="Aptoide.TextView.Regular.L.Theme">
+    <item name="android:textSize">19sp</item>
```

Comment:
You don't need the text sizes here, they are inherited from "Aptoide.TextView.Regular.L". Look at "Aptoide.TextView.Medium.M.BlackAlpha" for reference

Reply:
Done